src/Pulp/Run.purs

```diff
@@ -18,13 +14,16 @@ import Node.Encoding (Encoding(UTF8))
 import Node.FS.Aff as FS
 import Node.Path as Path
 import Node.Process as Process
+import Pulp.Args (Action(..))
+import Pulp.Args.Get (getOption')
 import Pulp.Build as Build
+import Pulp.Exec (exec)
 import Pulp.System.Files (openTemp)
 
 action :: Action
 action = Action \args -> do
   let opts = Map.union args.globalOpts args.commandOpts
-  out <- getOutputter args
+  -- out <- getOutputter args
```

Comment:
Maybe just remove this line rather than commenting it out?

Reply:
Done.